### DISCUSSION TITLE
Documentation for Assert\Optional

### DISCUSSION THREAD

**Discussion by kAlvaro** [symfony/symfony#58440] [Q&A] [answered]
I've deeply ashamed to ask this but... Where in [Symfony documentation](https://symfony.com/doc/current/index.html) can I find  `Assert\Optional`? Either it's a too common English word for search engines or it isn't documented at all, but I can't tell which one is the case.

**Comment by norkunas** [comment, marked as answer]
https://symfony.com/doc/current/reference/constraints/Collection.html#required-and-optional-field-constraints
